appserver/tests/admin/tests/src/test/java/org/glassfish/main/admin/test/OSGiCommandsITest.java

```diff
@@ -56,13 +57,15 @@ public class OSGiCommandsITest {
     @BeforeAll
     public static void waitOsgiReady() throws Exception {
         long timeout = System.currentTimeMillis() + 10_000L;
-        while (System.currentTimeMillis() < timeout) {
-            AsadminResult result = ASADMIN.exec("osgi", "lb");
-            if (!result.isError()) {
-                return;
+        for (int i = 0; i < 50; i++) {
```

Comment:
The while loop should be enough.

Reply:
>The while loop should be enough.

Good comment, but it's not a new thing. It was already there.